src/components/document/annotation-layer.tsx

```diff
@@ -76,52 +76,59 @@ export const AnnotationLayer = observer(function AnnotationLayer({
     }
   };
 
-  // Returns a function that will translate a point so it can be passed as a parameter to AnnotationButton
-  function getTranslateTilePointToScreenPoint(rowId: string, tileId: string) {
-    const tileBorder = 3;
+  // Returns the x and y offset of the top left corner of a tile with respect to the document
+  function getTileOffset(rowId: string, tileId: string): Point | undefined {
+    const tileBorder = 2;
 
     const rowElement = getRowElement(rowId);
-    if (!rowElement) return undefined;
+    if (!rowElement) return;
 
     const tileSelector = `[data-tool-id='${tileId}']`;
     const tileElements = canvasElement?.querySelectorAll(tileSelector);
     const tileElement = tileElements && tileElements.length === 1 ? tileElements[0] as HTMLElement : undefined;
-    if (!tileElement) return undefined;
+    if (!tileElement) return;
 
-    return (point: Point): Point | undefined => {
-      const [x, y] = point;
-      const _x = rowElement.offsetLeft + tileElement.offsetLeft - tileElement.scrollLeft
-        + x + tileBorder - (documentScrollX ?? 0);
-      const _y = rowElement.offsetTop + tileElement.offsetTop - tileElement.scrollTop
-        + y + tileBorder - (documentScrollY ?? 0);
-      return [_x, _y];
-    };
+    const x = rowElement.offsetLeft + tileElement.offsetLeft - tileElement.scrollLeft
+      + tileBorder - (documentScrollX ?? 0);
+    const y = rowElement.offsetTop + tileElement.offsetTop - tileElement.scrollTop
+      + tileBorder - (documentScrollY ?? 0);
+    return [x, y];
   }
 
+  // Returns an object bounding box with respect to the containing tile
   function getObjectBoundingBox(
-    rowId: string, tileId: string, objectId: string, objectType?: string
+    tileId: string, objectId: string, objectType?: string
   ) {
     const tileApi = tileApiInterface?.getTileApi(tileId);
     const objectBoundingBox = tileApi?.getObjectBoundingBox?.(objectId, objectType);
-    if (!objectBoundingBox) return undefined;
+    if (!objectBoundingBox) return;
+
+    return objectBoundingBox;
```

Comment:
If `undefined` is the only falsy value to be concerned about, this can be simplified to:
```typescript
    return tileApi?.getObjectBoundingBox?.(objectId, objectType);
```